elastalert/elastalert.py

```diff
@@ -875,7 +876,9 @@ def init_rule(self, new_rule, new=True):
             if prop not in rule:
                 continue
             new_rule[prop] = rule[prop]
-
+                   
+        self.scheduler.add_job(self.handle_rule_execution, 'interval', args=[new_rule] ,seconds=new_rule['run_every'].total_seconds(),id=new_rule['name'])
```

Comment:
Style: Add spaces after commas

Also, lets add max_instances=1, which should prevent overlapping runs if execution time goes over run_every